<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialClass</class>
 <widget class="QMainWindow" name="SerialClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口测试</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialPortLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="waitResponseLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>等待响应时间</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="waitResponseSpinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QComboBox" name="serialPortComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="requestLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送命令显示</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="requestLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>who are you?</string>
    </property>
   </widget>
   <widget class="QLabel" name="trafficLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>351</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string>No traffic.</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: Not running.</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="showDate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sendCMDBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发送命令CMD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收命令显示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="serial.qrc"/>
 </resources>
 <connections/>
</ui>
